src/main/java/org/betonquest/betonquest/conversation/SlowTellrawConvIO.java

```diff
@@ -58,11 +80,14 @@ public void run() {
                     displayText();
 
                     // Display endLines
-                    for (final String message : endLines) {
-                        SlowTellrawConvIO.super.print(message);
+                    if (endLines != null) {
+                        for (final String message : endLines) {
+                            SlowTellrawConvIO.super.print(message);
+                        }
                     }
```

Comment:
```suggestion
                    for (final String message : endLines) {
                        SlowTellrawConvIO.super.print(message);
                    }
```
Why do we need this check now, if it is now not possible to answer before everything is printed?